.cirrus.yml

```diff
@@ -5,135 +5,96 @@ env:
-
-# Only github users with write-access can define or use encrypted variables
-# This credential represents a service account with access to manage both VMs
-# and storage.
 gcp_credentials: ENCRYPTED[13e51806369f650e6ccc326338deeb3c24052fc0a7be29beef2b96da551aed3200abbb6c6406a936bb4388fb2758405c]
 
 # Default compute settings unless overridden within tasks (below)
-gce_instance: {"image_project": "conmon-222014", "zone": "us-central1-f", "cpu": 4, "memory": "16Gb", "disk": 200, "image_name": "no-image-specified-in-task"}
+gce_instance:
+    image_project: "${IMAGE_PROJECT}"
+    zone: "us-central1-f"
+    cpu: 4
+    memory: "16Gb"
+    disk: 200  # MANDATORY 200GB for obscure less-documented performance reasons
+    image_name: "${FEDORA_CACHE_IMAGE_NAME}"
 
-# Default timeout for each task
-timeout_in: '120m'
 
-# Every *_task runs in parallel in separate VMs. The name prefix only for reference
-# in WebUI, and will be followed by matrix details.  This task runs the integration
-# testing for every platform
-cri-o_integration_task:
+# Update metadata on VM images referenced by this repository state
+# N/B: This is a _CRITICAL_ task, VM Image timestamps indicate which are
+# candidates for inactivity-pruning.  It is assumed to run execute periodically.
+meta_task:
 
-    depends_on:
-        - 'config'
-        - 'fmt'
+    # see bors.toml
+    skip: $CIRRUS_BRANCH =~ ".*\.tmp"
 
-    gce_instance:
-        # Generate multiple parallel tasks, covering all possible
-        # 'matrix' combinations.
-        matrix:
-            # Images are generated separately, from build_images_task (below)
-            image_name: "${FEDORA_CACHE_IMAGE_NAME}"
+    container:
+        image: "quay.io/libpod/imgts:latest"  # from libpod repo.
+        cpu: 1
+        memory: 1
 
     env:
-        matrix:
-            TEST_USERNS: 0
-            TEST_USERNS: 1
+        # Space-separated list of images used by this repository state
+        IMGNAMES: |-
+            ${FEDORA_CACHE_IMAGE_NAME}
+            ${PRIOR_FEDORA_CACHE_IMAGE_NAME}
+            ${UBUNTU_CACHE_IMAGE_NAME}
+            ${PRIOR_UBUNTU_CACHE_IMAGE_NAME}
+        BUILDID: "${CIRRUS_BUILD_ID}"
+        REPOREF: "${CIRRUS_CHANGE_IN_REPO}"
+        GCPJSON: ENCRYPTED[08de2c74178470b1bc85a107e9962f06dbd11d33c7adf024d3e48ae4399ca5383f9d3ad0e2fd65c3ce12750dd6ef8803]
+        GCPNAME: ENCRYPTED[561ce33a9357e5b8e3fb54739c3af31730c0c3b736792f16a67026a8544379d83ff3c27d6fea1c7797a6ae49b6e58115]
+        GCPPROJECT: ENCRYPTED[1992c55f028f9a0d03f01f84d173425a222ad52bcecdc97d29bc99bd36152fadadd2bad7f79498307b2ce85409fb4bc8]
+        CIRRUS_CLONE_DEPTH: 1  # source not used
 
-    setup_environment_script: '$SCRIPT_BASE/setup_environment.sh'
-    integration_test_script: '$SCRIPT_BASE/cri-o_test.sh'
+    # Source not used
+    clone_script: mkdir -p '${CIRRUS_WORKING_DIR}'
 
-# this task runs the conmon integration tests
-integration_task:
+    script: '/usr/local/bin/entrypoint.sh |& ${_TIMESTAMP}'
```

Comment:
This cirrus.yml does not have `_TIMESTAMP`

Reply:
oh thanks!